IFC4x3/Quantities/d/Depth_3RJBqFyPr5BuHqJx2YiXLV/DocQuantity.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocQuantity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Depth_3RJBqFyPr5BuHqJx2YiXLV" Name="Depth" UniqueId="db4cbd0f-f19d-452f-8474-4fb0a2b2155f" />
```

Comment:
There is no harmonize depth quantities. 
Quantity 1 definition : "Depth (or thickness) of the opening, in case of openings it shall be identical to the width (or thickness) of the voided element, in case of recesses it shall be less. Only provided, if the depth is constant."
Quantity 2 definition : "Depth (one direction of the non-projected foot print area) of the slab. It shall only be provided, if the slab is rectangular.
> NOTE&nbsp; Also refered to as width, but not to be confused with the "Width" quantity, that denotes the thickness in the context of the slab."

Reply:
So I keep this new quantity, and add it directly to DistributionChamberBasedQuantitySet. I have also erased the Qset from raildomain. Commit : c3c62d498a2e1bfdaee4063919ad0d65a9d9f806